cli/waiter/token_post.py

```diff
@@ -69,7 +69,26 @@ def create_or_update_token(clusters, args, _, action):
     """Creates (or updates) a Waiter token"""
     guard_no_cluster(clusters)
     logging.debug('args: %s' % args)
-    token_name = args.pop('token')
+    token_name_from_args = args.pop('token', None)
+    json_file = args.pop('json', None)
+    if json_file:
+        if len(args) > 0:
+            raise Exception('You cannot specify both a token JSON file and token field flags at the same time.')
+
+        token_fields = load_json_file(json_file)
+        if not token_fields:
+            raise Exception(f'Unable to load token JSON from {json_file}.')
+    else:
+        token_fields = args
+
+    token_name_from_json = token_fields.pop('token', None)
+    if token_name_from_args and token_name_from_json:
+        raise Exception('You cannot specify the token name both as an argument and in the token JSON file.')
```

Comment:
Should we allow it if it is the same value?

Reply:
Never mind, we have similar behavior in Waiter routers where we allow only the query param or the json payload.